app/src/main/res/layout/calculator_button.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <Button
+        android:id="@+id/centerButton"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:text=""
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/topLeftText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:rotation="0"
+        android:text=""
+        android:translationZ="90dp"
+        app:layout_constraintStart_toStartOf="@+id/centerButton"
+        app:layout_constraintTop_toTopOf="@+id/centerButton" />
+
+    <TextView
+        android:id="@+id/topRightText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text=""
+        android:translationZ="90dp"
+        app:layout_constraintEnd_toEndOf="@+id/centerButton"
+        app:layout_constraintTop_toTopOf="@+id/centerButton" />
+
+    <TextView
+        android:id="@+id/bottomRightText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text=""
+        android:translationZ="90dp"
+        app:layout_constraintBottom_toBottomOf="@+id/centerButton"
+        app:layout_constraintEnd_toEndOf="@+id/centerButton" />
+
+    <TextView
+        android:id="@+id/bottomLeftText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text=""
+        android:translationZ="90dp"
```

Comment:
90dp seems a bit much. Isn't Android using something like 4dp or 8dp usually?

Reply:
Fixed in [1b88e95](https://github.com/mr-kenhoff/qalculate-android/pull/31/commits/1b88e95f750d7eea3239524c4d8de43efc11a176)